docs/integrations/data-ingestion/azure-synapse/index.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_label: 'Azure Synapse'
+slug: /integrations/azure-synapse
+description: 'Introduction to Azure Synapse with ClickHouse'
+keywords: ['clickhouse', 'azure synapse', 'azure', 'synapse', 'microsoft', 'azure spark', 'data']
+title: 'Integrating Azure Synapse with ClickHouse'
+---
+
+import TOCInline from '@theme/TOCInline';
+import Image from '@theme/IdealImage';
+import sparkConfigViaNotebook from '@site/static/images/integrations/data-ingestion/azure-synapse/spark_notebook_conf.png';
+import sparkUICHSettings from '@site/static/images/integrations/data-ingestion/azure-synapse/spark_ui_ch_settings.png';
+
+# Integrating Azure Synapse with ClickHouse
+
+[Azure Synapse](https://azure.microsoft.com/en-us/products/synapse-analytics) is an integrated analytics service that combines big data, data science and warehousing to enable fast, large-scale data analysis.
+Within Synapse, Spark pools provide on-demand, scalable [Apache Spark](https://spark.apache.org) clusters that let users run complex data transformations, machine learning, and integrations with external systems.
+
+This article will show you how to integrate the [ClickHouse Spark connector](/integrations/apache-spark) when working with Apache Spark within Azure Synapse.
+
+
+<TOCInline toc={toc}></TOCInline>
+
+## Add the connector's dependencies {#add-connector-dependencies}
+Azure Synapse supports three levels of [packages maintenance](https://learn.microsoft.com/en-us/azure/synapse-analytics/spark/apache-spark-azure-portal-add-libraries):
+1. Default packages
+2. Spark pool level
+3. Session level
+
+<br/>
+
+Follow the [Manage libraries for Apache Spark pools guide](https://learn.microsoft.com/en-us/azure/synapse-analytics/spark/apache-spark-manage-pool-packages) and add the following required dependencies to your Spark application
+   - `clickhouse-spark-runtime-{spark_version}_{scala_version}-{connector_version}.jar` - [official maven](https://mvnrepository.com/artifact/com.clickhouse.spark)
+   - `clickhouse-jdbc-{java_client_version}-all.jar` - [official releases](https://github.com/ClickHouse/clickhouse-java/releases)
+
+Please visit our [Spark Connector Compatibility Matrix](/integrations/apache-spark/spark-native-connector#compatibility-matrix) docs to understand which versions suit your needs.
```

Comment:
not related to the docs, just a note: JDBC version in Spark connector is quite outdated 😞